Flint/src/main/java/CLI_Adapter/CLIAdapter.java

```diff
@@ -0,0 +1,67 @@
+package CLI_Adapter;

```

Comment:
Personally this seems like something that should be in `package flint`.
We don't need a separate folder for every class and this logically belongs in that package (in my opinion)

Reply:
Moved to flint folder, reflected update in google doc spec